src/Math-Tests-Numerical/PMLeastSquaresTest.class.st

```diff
@@ -0,0 +1,125 @@
+"
+A PMLeastSquaresTest is a test class for testing the behavior of PMLeastSquares
+"
+Class {
+	#name : #PMLeastSquaresTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'leastSquares'
+	],
+	#category : #'Math-Tests-Numerical'
+}
+
+{ #category : #running }
+PMLeastSquaresTest >> setUp [
+	super setUp.
+	leastSquares := PMLeastSquares new.
+]
+
+{ #category : #tests }
+PMLeastSquaresTest >> testPseudoinverseOfDiagonalSquareMatrix [
+
+	| matrix expectedInverse inverse |
+	
+	matrix := PMMatrix rows: #(
```

Comment:
Consider consistency in formatting the rows

Reply:
I did a new commit formatting the rows